scripts/dts/gen_defines.py

```diff
@@ -165,6 +176,12 @@ def augment_node(node):
         alias_idents.append(f"{compat_s}_{alias_ident}")
     node.z_alias_idents = alias_idents
 
+    # Add z_label_idents, which are formed from devicetree labels
+    # for the node. This is distinct from the label property
+    # used by zephyr as an argument for device_get_binding().
+    node.z_label_idents = [f'NODELABEL_{str2ident(label)}'
```

Comment:
Did you not put this in z_other_idents on purpose? The comment for this function is wrong now

Reply:
I wasn't sure the implication of putting it in z_other_idents, if we'd end up generating stuff like DT_NODELABEL_GPIOA_BASE_ADDRESS_0 or not?  I wanted to avoid that.

Reply:
Why do you want to avoid that? If you do, we probably need to refactor for_each_ident().